### PR TITLE
Fix Chat UI/UX #2354, #2355, #2353

### LINKED ISSUE #2354: Fix SUSI Chat in Mobile view
**Actual Behaviour**

Buggy

**Expected Behaviour**

Make it mobile compatible

**Screenshots of the issue**

Screenshot 2019-06-23 at 3 49 49 AM

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by fragm3**
Fixes #2354, #2355, #2353, #2362

Changes: 
- Fix Chat Search highlight
- Fix SUSI Chat in Mobile view
- Fix SkillRating Popover, Add styled component
- Fix console errors in chat

Demo Link: https://pr-2356-fossasia-susi-web-chat.surge.sh

Screenshots of the change:

Screenshot 2019-06-23 at 9 17 44 PM

Screenshot 2019-06-23 at 9 33 41 PM

Screenshot 2019-06-23 at 9 00 40 PM

### CODE REVIEW

File: src/components/ChatApp/MessageListItem/generateMessageBubbleUtil.js [hunk 1]
```diff
@@ -16,6 +16,7 @@ import {
 } from './helperFunctions.react.js';
 import styled, { css } from 'styled-components';
 import MessageBubble from './MessageBubbleStyle';
+import './highlight.css';
```
[the remark this answers is not in the document]
Comment (fragm3) [reply src/components/ChatApp/MessageListItem/generateMessageBubbleUtil.js]: Yes, for styling the marked keyword. Tried other ways didn't worked

**Comment by fragm3** [comment]
@akshatnitd fixed the colors:

Screenshot 2019-06-24 at 1 07 57 PM

**Comment by akshatnitd** [comment]
@fragm3 Fix the searchbar, rest looks good.

**Comment by fragm3** [comment]
@akshatnitd made the changes:

Iphone 8 plus:
Screenshot 2019-06-24 at 8 53 28 PM

Iphone 8:
Screenshot 2019-06-24 at 8 53 44 PM

**Pull request merged**